kubernetes/resource_kubernetes_horizontal_pod_autoscaler_v2beta2_test.go

```diff
@@ -129,6 +129,48 @@ func TestAccKubernetesHorizontalPodAutoscalerV2Beta2_basic(t *testing.T) {
 	})
 }
 
+func TestAccKubernetesHorizontalPodAutoscalerV2Beta2_containerResource(t *testing.T) {
```

Comment:
This test will fail on clusters where the `HPAContainerMetrics` feature gate is not enabled. Right now there isn't a reliable way to check if a feature gate has been enabled programmatically. You have to check the `--feature-gates` args supplied to the apiserver, which we can't even do on EKS, GKE, et al. 

For the moment I would add a `t.Skip()` to this test with a message explaining this. We can remove when the Featuer gate becomes defaulted to true.

Reply:
I have added `ErrorCheck` with `t.Skip()`. In this case, the step will be executed and skipped only if it fails.